src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -160,4 +164,31 @@ public function get_limit_options() {
 
 		return $options;
 	}
+
+	/**
```

Comment:
* :no_entry_sign: Doc comment for parameter "$arguments" missing
* :no_entry_sign: Doc comment for parameter "$context" missing

<sub>hash: 82f4604f7e1788c54dacbc033cfda794</sub>